tests/integration/models/reddit/test_subreddit.py

```diff
@@ -9,6 +11,45 @@
 
 class TestSubreddit(IntegrationTest):
     @mock.patch('time.sleep', return_value=None)
+    def test_create(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette('TestSubreddit.test_create'):
+            # These values are hard-coded for the benefit of the cassettes.
+            new_name = 'PRAW_cvolgqjmge'
+            nonexistent_subreddit = 'PRAW_gbchyuppzp'
+
+            # Extra kwargs that are static throughout the test
+            kwargs = {'subreddit_type': 'public',
+                      'wikimode': 'disabled'}
+
+            with pytest.raises(Redirect) as excinfo:
```

Comment:
Do we need this request? If at the time we're testing create and the subreddit happens to exist, we can update the code to create one that doesn't exist.